src/action.js

```diff
@@ -11,7 +11,11 @@ var geo_action = {
   select: 'geo_action_select',
   unzoomselect: 'geo_action_unzoomselect',
   zoom: 'geo_action_zoom',
-  zoomselect: 'geo_action_zoomselect'
+  zoomselect: 'geo_action_zoomselect',
+
+  // annotation actions
+  annotation_polygon: 'geo_annotation_polygon',
```

Comment:
I was viewing this as the action belongs to the annotation module and the action is to make a polygon.